src/main/java/org/blockchain_innovation/factom/identiy/did/request/CreateFactomDidRequest.java

```diff
@@ -0,0 +1,126 @@
+package org.blockchain_innovation.factom.identiy.did.request;
+
+import org.blockchain_innovation.factom.identiy.did.DIDVersion;
+import org.blockchain_innovation.factom.identiy.did.entry.CreateFactomDIDEntry;
+import org.factomprotocol.identity.did.model.DidMethodVersion;
+import org.factomprotocol.identity.did.model.FactomDidContent;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CreateFactomDidRequest {
+    private final DIDVersion didVersion;
+    private final List<CreateKeyRequest> managementKey;
```

Comment:
I am guessing the names are not plural because we cannot rely on Json libs for annotations that define the json property name right?

Reply:
I think I can make these plural without any complications, as we convert them to `FactomDidContent` objects before any JSON stuff. I was just trying to keep the request models in line with the did models. Is there a reason these are singular in the on-chain models?

Reply:
The reason is that we have singular in the Factom DID spec for the entryData https://github.com/factom-protocol/FIS/blob/master/FIS/DID.md#entry-structure, which is in line with the W3C spec also being singular.

We currently cannot rely on annotations in the properties to fix the keys during serialization, because we are JSON library agnostic to limit the amount of deps and let the library user decide. Given this is not communicated on chain I guess we can make these plural in here

Reply:
Ok, updated.